src/Profiles/Lev/Display/ANTPLUS_ProfileLevDisplay.cpp

```diff
@@ -0,0 +1,119 @@
+#include <Profiles/Lev/Display/ANTPLUS_ProfileLevDisplay.h>
+#include <Profiles/Lev/ANTPLUS_LevPrivateDefines.h>
+#include <CommonDataPages/RX/ANTPLUS_ModeSettings.h>
+#include <CommonDataPages/ANTPLUS_CommonDataPagePrivateDefines.h>
+
+ProfileLevDisplay::ProfileLevDisplay() : BaseSlaveProfile() {
+    setChannelConfig();
+}
+
+ProfileLevDisplay::ProfileLevDisplay(uint16_t deviceNumber) : BaseSlaveProfile(deviceNumber) {
+    setChannelConfig();
+}
+
+ProfileLevDisplay::ProfileLevDisplay(uint16_t deviceNumber, uint8_t transmissionType) : BaseSlaveProfile(deviceNumber, transmissionType) {
+    setChannelConfig();
+}
+
+void ProfileLevDisplay::setChannelConfig() {
+    setChannelType(ANTPLUS_LEV_CHANNELTYPE);
+    setDeviceType(ANTPLUS_LEV_DEVICETYPE);
+    setChannelPeriod(ANTPLUS_LEV_CHANNELPERIOD);
+    setSearchTimeout(ANTPLUS_LEV_SEARCHTIMEOUT);
+}
+
+void ProfileLevDisplay::onBroadcastData(BroadcastData& msg) {
+    BaseProfile::onBroadcastData(msg);
+    LevBaseMainDataPage dp = LevBaseMainDataPage(msg);
+    if (!handleDataPage(dp)) {
+        callOnOtherDataPage(msg);
+    }
+}
+
+void ProfileLevDisplay::onAcknowledgedData(AcknowledgedData& msg) {
+    BaseProfile::onAcknowledgedData(msg);
+    LevBaseMainDataPage dp = LevBaseMainDataPage(msg);
+    if (!handleDataPage(dp)) {
+        callOnOtherDataPage(msg);
+    }
+}
+
+/*
+void ProfileLevDisplay::onChannelEventResponse(ChannelEventResponse& msg) {
```

Comment:
More debug?

Reply:
Removed. I made this just to show a potential user how to receive channel event notifications.

Reply:
Ah i would make an example for that, I wouldn't expect anyone to dive into the backend to find that